dispatch/apps/events/views.py

```diff
@@ -0,0 +1,50 @@
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+
+from dispatch.apps.events.facebook import FacebookEvent
+from dispatch.apps.events.forms import EventForm
+from dispatch.apps.events.models import Event
+
+def success(request):
+
+    return HttpResponse('Your event has been created')
+
+def submit(request):
+
+    if request.POST.get("submit_facebook") and (request.POST.get('facebook_url') is not None):
```

Comment:
Use a variable to prevent calling `request.POST.get("facebook_url")` multiple times:

```py

facebook_url = request.POST.get('facebook_url', None)

if request.POST.get("submit_facebook") and facebook_url:
    ...